freud/plot.py

```diff
@@ -142,20 +196,75 @@ def pmft_plot(pmft, ax=None):
         ax.yaxis.set_ticks([i for i in range(int(ylims[0]), int(ylims[1]+1))])
         ax.set_xlabel(r'$x$')
         ax.set_ylabel(r'$y$')
-
         ax.set_title('PMFT')
+
+        ax_divider = make_axes_locatable(ax)
+        cax = ax_divider.append_axes("right", size="7%", pad="10%")
+
         im = ax.imshow(np.flipud(pmft_arr),
                        extent=[xlims[0], xlims[1], ylims[0], ylims[1]],
                        interpolation='nearest', cmap='viridis',
                        vmin=-2.5, vmax=3.0)
 
-        ax_divider = make_axes_locatable(ax)
-        cax = ax_divider.append_axes("right", size="7%", pad="10%")
-        cb = colorbar(im, cax=cax)
-        cb.set_label_text(r"$k_B T$")
+        cb = Colorbar(cax, im)
+        cb.set_label(r"$k_B T$")
+
         return ax
 
 
+def plot_density(density, box, ax=None):
+    R"""Helper function to plot density diagram.
+
+    Args:
+        density (:math:`\left(N_x, N_y\right)` :class:`numpy.ndarray`):
+            Array containing density.
+        box (:class:`freud.box.Box`):
+            Simulation box.
+        ax (:class: matplotlib.Axes.axes): axes object to plot on.
+            If :code:`None`, make a new axes and figure object.
+            (Default value = :code:`None`).
+
+    Returns:
+        :class: matplotlib.Axes.axes: axes object with the diagram.
+    """
+    if not MATPLOTLIB:
+        return None
+    try:
+        from mpl_toolkits.axes_grid1.axes_divider import make_axes_locatable
+        from matplotlib.colorbar import Colorbar
+        import matplotlib.transforms
+    except ImportError:
+        return None
+
+    if ax is None:
+        fig = Figure()
+        ax = fig.subplots()
+
+    xlims = (-box.Lx/2, box.Lx/2)
+    ylims = (-box.Ly/2, box.Ly/2)
+
+    ax.set_title('Gaussian Density')
+    ax.set_xlabel(r'$x$')
+    ax.set_xlabel(r'$x$')
+
+    ax_divider = make_axes_locatable(ax)
+    cax = ax_divider.append_axes("right", size="7%", pad="10%")
+
+    im = ax.imshow(density, extent=[xlims[0], xlims[1], ylims[0], ylims[1]])
+
+    transform = matplotlib.transforms.Affine2D().skew(box.xy, 0)
```

Comment:
For now, I don't think we want to transform this. In the future I'd like to fix the behavior of GaussianDensity and then skew it accordingly, see #319. Please just comment out the code that shears the image for now, and document that it's related to issue #319.